source/guides/persistence.html.erb

```diff
@@ -5,43 +5,53 @@ title: Elixir Girls Guides Slackir App adding persistence
 <p>So now we have a basic chat app but once our session is closed all of our messages are lost. If we were able to create, read, update and delete from a database we would be able to maintain a history of messages or have persistence across our application.</p>
 
 <h2>Creating our Database</h2>
-<p>First, we’ll create and run a migration. Migrations are pieces of code that create or change database columns. In Phoenix, we can invoke mix phoenix.gen.model from the terminal to generate our model, then run the migration:</p>
+
+<p>First, we’ll create a migration and some code to handle persistence. Migrations are pieces of code that create or change database columns. In Phoenix, we can invoke <code>mix phx.gen.context</code> from the terminal to generate a context with functions, schema and migration:</p>
+
+<% code("bash") do %>
+$ mix phx.gen.context Conversations Message messages name:string message:string
+<% end %>
+
+<p>Explanation on that command:</p>
+<ul>
+  <li><b>Context: </b><code>Conversations</code> - The context is an Elixir module that serves as an boundary for the given
+resources like messages. A context often holds many related resources.</li>
+  <li><b>Schema: </b><code>Message</code> - The schema is responsible for mapping the database fields into an Elixir struct.</li>
+  <li><b>Schema table: </b><code>messages</code> - The schema table is just a schema plural name for table in database.</li>
+  <li><b>Fields </b><code>name:string</code> and <code>message:string</code>- The fields are columns in database table where <code>message</code> is a filed name and <code>string</code> is field type in <code>message:string</code>.</li>
```

Comment:
Maybe we can link here to a list of available types in Ecto?

Reply:
Good idea, I will add it.